### ISSUE TITLE
zowe-copy-to-JES.sh should not create its own log file

### ISSUE THREAD

**Issue by OnnoVdT** [zowe/zowe-install-packaging#1548]
**Describe the bug**
zowe-copy-to-JES.sh is a utility called by other scripts. Therefore it should append the log of the calling script instead of creating its own, scattering the log data.

**To Reproduce**
invoke scripts/utils/zowe-install-xmem.sh

**Expected behavior**
zowe-copy-to-JES.sh should accept a parm to use a given log file. An example of this is in install/zowe-install.sh

**Comment by stevenhorsman** [comment]
Hi @OnnoVdT , I'm going to need more info on this. I implemented this several months ago under #1280 and looking at the nightly tests it seems to be working, showing the the zowe-copy-to-jes log in inlined in the install-xmem..
zowe-install-xmem-2020-07-31-06-20-14.log:
```
...
listds 'ZOWEAD3.SMPE.SZWESAMP' members
      ZWESIP00
    Copying SAMPLIB member ZWESIP00
    Info:  member ZWESIP00 has been successfully copied to dataset ZOWEAD3.PARMLIB
    <zowe-copy-to-JES.sh>
    started from /ZOWE/staging/usr/lpp/zowe/scripts/utils
    userid  = ZOWEAD3
    templib = ZOWEAD3.zowetemp.instproc.SZWESAMP.proclib
...
```

**Comment by OnnoVdT** [comment]
My bad, I must have been getting tired. Looking at the script again I see the -f option, and zowe-install-xmem using it. I clearly misinterpreted something back then.
